### DISCUSSION TITLE
receita de bolo

### DISCUSSION THREAD

**Discussion by ghost** [community/community#125182] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Massa
óleo
1/2 xícara (chá) de óleo
cenoura
3 cenouras médias raladas
ovo
4 ovos
açúcar
2 xícaras (chá) de açúcar
farinha de trigo
2 e 1/2 xícaras (chá) de farinha de trigo
fermento em pó químico
1 colher (sopa) de fermento em pó
Cobertura
manteiga
1 colher (sopa) de manteiga
chocolate em pó
3 colheres (sopa) de chocolate em pó
açúcar
1 xícara (chá) de açúcar
leite
1 xícara (chá) de leite

**Comment by cixpy** [comment, marked as answer]
farei com certeza
